Week2/homework-midy/lemonAllergy.js

```diff
@@ -0,0 +1,22 @@
+const fruitBasket = [
+  'Apple',
+  'Lemon',
+  'Grapefruit',
+  'Lemon',
+  'Banana',
+  'Watermelon',
+  'Lemon',
+  'Orange',
+];
+
+function getRidOfLemons(basket) {
+  const noLemons = basket.filter(fruit => fruit !== 'Lemon');
+  const numberedFruit = noLemons.map(i => {
+    i = i.toLowerCase();
+    i = i + 's';
+    return i;
```

Comment:
Here you can reduce the LOC (Lines Of Code) by doing something like this
`const numberedFruit = noLemons.map(i => i.toLowerCase() + 's')`
note that the above is the same as 
```js
const numberedFruit = noLemons.map(i => {
    return i.toLowerCase() + 's')
}
```
in an arrow function you can return the result in one line. Like you did with the filter function above

Reply:
Oh, I didn't think of that!